native/core/src/events/common/categories.rs

```diff
@@ -1,30 +1,10 @@
 use super::Display;
 use derive_builder::Builder;
-use ruma_events::{EventContent, PossiblyRedactedStateEventContent, StateEventType};
-use ruma_macros::EventContent;
+use matrix_sdk_base::ruma::events::macros::EventContent;
 use serde::{Deserialize, Serialize};
 
-/// The possibly redacted form of [`CategoriesEventContent`].
-///
-/// This type is used when it's not obvious whether the content is redacted or not.
-#[derive(Clone, Debug, Deserialize, Serialize)]
-#[allow(clippy::exhaustive_structs)]
-pub struct PossiblyRedactedCategoriesStateEventContent();
```

Comment:
We don't require this with new sdk upgrade?

Reply:
No, that workaround was needed for an older ruma version, but it works properly now with the latest `ruma_event` macro.